lcls-twincat-motion/Library/POUs/Motion/States/FB_PositionStateMove.TcPOU

```diff
@@ -114,15 +122,24 @@ fbMotionRequest(
     bBusy => bBusy,
     bDone => bDone);
 
+rtReset(CLK:=bReset);
+IF rtReset.Q THEN
+    nLatchAllowErrorID := 0;
+END_IF
+
 // Inject custom error if we can't move because of bMoveOk or bValid
-IF bExecute AND NOT bAllowMove THEN
+IF nLatchAllowErrorID <> 0 OR (bExecute AND NOT bAllowMove) THEN
```

Comment:
In the scenario of `nLatchAllowErrorID <> 0`, it's necessary to redo this `IF` block not to re-look things up, but ensure that `bError` is set (no matter what `fbMotionRequest` does to it), is that right? Trying to follow the logic, and it seems largely a no-op beside for that.

Reply:
That's exactly correct. I need to reapply my own FB's error status, including `bError` and `nErrorID`, irrespective of the internal error state of `fbMotionRequest`.